### ISSUE TITLE
Accessibility: Status images in the delegate table are not keyboard accessible

### ISSUE THREAD

**Issue by arkmoon** [ArkEcosystem/explorer#693]
**Expected Behavior**

A screen reader, or even a sighted user who is only using a keyboard, should be able to understand the "status" column in the main Delegate table here:

https://dexplorer.ark.io/delegate-monitor

**Current Behavior**

The current popover for Status seems to be an mouse-only 'on-hover' event, which is not accessible for keyboard-only users, including the blind and those who are unable to use a mouse. The image itself does not contain any alt text, and a screen reader simply says "image" when navigated to, and the popover is not triggered.

**Possible Solution**

Please either add some alt text that mimics what's in the popover, or trigger the popover on-focus on that image in the table so the screen reader can announce it. Example, include's aria-role and aria-describedBy attributes:
https://heydonworks.com/practical_aria_examples/#input-tooltip

**Comment by ArkEcosystemBot** [comment]
**Thanks for opening this issue!** A maintainer will review this in the next few days and explicitly select labels so you know what's going on.

_If no reviewer appears after a week, a reminder will be sent out._

**Comment by dated** [comment]
_stalebot, be gone!_

**Comment by ArkEcosystemBot** [comment]
This issue has been closed. If you wish to re-open it please provide additional information.
